src/elm/comparison.coffee

```diff
@@ -40,3 +40,12 @@ module.exports.GreaterOrEqual = class GreaterOrEqual extends Expression
     args = @execArgs(ctx).map (x) -> Uncertainty.from x
     return null unless args[0]? and args[1]?
     args[0].greaterThanOrEquals args[1]
+
+module.exports.SameAs = class SameAs extends Expression
+  constructor: (json) ->
+    super
+    @precision = json.precision
+
+  exec: (ctx) ->
+    [a, b] = @execArgs(ctx)
+    if a? and b? then a.sameAs(b, @precision?.toLowerCase()) else null
```

Comment:
I expect you moved `SameAs` from `datetime.coffee` to `comparison.coffee` because it now applies to both datetimes and intervals -- and comparison makes sense at a certain level.

That said, we've tried to organize these functions into files according to how the spec lists the operators in the appendix.  In the appendix, `SameAs` is not listed in the `Comparison` section (see: https://cql.hl7.org/09-b-cqlreference.html#comparison-operators-4) -- so this isn't where we'd normally put it.

If an operator is listed in more than one section (like in this case), we usually put it in the `overloaded.coffee` file.  For consistency, that's probably a better place for it than `comparison.coffee`.  Would you mind moving it there?

Reply:
Will do!